docs/publisher/index.rst

```diff
@@ -0,0 +1,59 @@
+Publisher Client
+================
+
+Publish operations are handled through the
+:class:`~.pubsublite.cloudpubsub.publisher_client.PublisherClient` class (aliased as
+``google.cloud.pubsublite.cloudpubsub.PublisherClient``).
+
+You should instantiate a publisher client using a context manager:
+
+.. code-block:: python
+
+    from google.cloud.pubsublite.cloudpubsub import PublisherClient
+
+    with PublisherClient() as publisher_client:
+        pass
+
```

Comment:
nit: "# use publisher_client" instead of "pass", so that people know that they have to use the client in the "with" block